src/TableRecordInstantiator.php

```diff
@@ -38,12 +38,6 @@ public function instantiate(Fixture $fixture)
         $tablename = substr($fixture->getClass(), 6);
         $r = new TableRecord($tablename);
 
-        if (!isset($this->ids[$tablename])) {
```

Comment:
@igormukhingmailcom maybe then we should tag this as 2.0.0? coz it's not backwards-compatible